tests/Unit/Facade.Tests/StockTriggerLevelsServiceTests/WhenAdding.cs

```diff
@@ -0,0 +1,69 @@
+﻿namespace Linn.Stores.Facade.Tests.StockTriggerLevelsServiceTests
+{
+    using FluentAssertions;
+
+    using Linn.Common.Facade;
+    using Linn.Stores.Domain.LinnApps.StockLocators;
+    using Linn.Stores.Resources;
+
+    using NSubstitute;
+
+    using NUnit.Framework;
+
+    public class WhenAdding : ContextBase
+    {
+        private readonly int stockTriggerLevelId = 1;
+
+        private StockTriggerLevelsResource resource;
+
+        private IResult<StockTriggerLevel> result;
+
+        [SetUp]
+        public void SetUp()
+        {
+            var stockTriggerLevel = new StockTriggerLevel
+            {
+                PartNumber = "CTC PART",
+                Id = 1,
+                KanbanSize = 1,
+                LocationId = 123,
+                MaxCapacity = 1,
+                PalletNumber = 1,
+                TriggerLevel = 1
+            };
+
+            this.resource = new StockTriggerLevelsResource
+            {
+                PartNumber = "ES PART",
+                Id = 1,
+                KanbanSize = 2,
+                LocationId = 123,
+                MaxCapacity = 12,
+                PalletNumber = 456,
+                TriggerLevel = 400
+            };
+
+            this.result = this.Sut.Add(this.resource);
+        }
+
+        [Test]
+        public void ShouldAdd()
+        {
+            this.Repository.Received().Add(Arg.Any<StockTriggerLevel>());
```

Comment:
What are we actually testing here? the fact that the repository is called and that the Created varaint of the IResult are returned are just functions of the common library code - you shouldn't be unit testing them here. If you want to test the /create route an integration test would be a better place to do so

Reply:
Fair point. It's a fairly standard Add so I suppose I could take this out.